### PR TITLE
make sure omitted terms in ontology mapping are discoverescase-sensitive

### PR DESCRIPTION
**Pull request by le-ander**
currently, "unknown" is the correct identifier for unknown ontology entries. as matching of these was not done case-sensitive, the term "Unknown" was skipped when the ontology mapping tsv was created. this led to an error when loading the dataset.

### CODE REVIEW

**Comment by le-ander** [comment]
@davidsebfischer FYI

File: sfaira/unit_tests/update_loaders/update_celltypes.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 SYMBOL_COL = "target"
 ID_COL = "target_id"
-UNKNOWN_IDS = ["UNKNOWN", "NOT_A_CELL"]
+unknown_IDS = ["unknown", "NOT_A_CELL"]
```
Comment (davidsebfischer) [inline sfaira/unit_tests/update_loaders/update_celltypes.py:13]: This list is a global constant and should be upper case I think?

**Review by davidsebfischer — COMMENTED** [review]
Thanks for pointing this out!

File: sfaira/unit_tests/update_loaders/update_celltypes.py (hunk 1 above)
Comment (le-ander) [reply sfaira/unit_tests/update_loaders/update_celltypes.py:13]: indeed, will fix!

**Pull request merged**